content/en/guides/integrations/skorch.md

```diff
@@ -8,7 +8,7 @@ title: Skorch
 weight: 400
 ---
 
-You can use Weights & Biases with Skorch to automatically log the model with the best performance, along with all model performance metrics, the model topology and compute resources after each epoch. Every file saved in `wandb_run.dir` is automatically logged to W&B servers.
+You can use W&B with Skorch to automatically log the model with the best performance, along with all model performance metrics, the model topology and compute resources after each epoch. Every file saved in `wandb_run.dir` is automatically logged to W&B servers.
```

Comment:
```suggestion
You can use W&B with Skorch to automatically log the model with the best performance, along with all model performance metrics, the model topology and compute resources after each epoch. Every file saved in `wandb_run.dir` is automatically logged to W&B.
```
Most of the time, we don't refer to "W&B servers". Usually, we just use "W&B". If we are specifically talking about a self-hosted instance of W&B, we instead refer to "W&B Dedicated Cloud" or "W&B Self-Managed".